kernel/lib/fabutil/errors.go

```diff
@@ -0,0 +1,23 @@
+package fabutil
```

Comment:
can we just call `fabutil` `util`?

Reply:
renamed. I try to avoid util/utils sometimes because package name collision is a bigger issue in golang and it often feels like every module has a util.